server/bin/pbench-server-put-shim.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+# Forward tar balls received via SSH from version 002 clients
+# to the configured pbench server using the pbench agent's new
+# PUT API interface.
+
+import os
+import sys
+import tempfile
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger
+from pbench.process_tb import ProcessTb
+from pbench.report import Report
+
+
+_NAME_ = "pbench-server-put-shim"
+
+
+def main(cfg_name: str) -> int:
+    if not cfg_name:
+        print(
+            f"{_NAME_}: ERROR: No config file specified; set"
+            " _PBENCH_SERVER_CONFIG env variable or use --config <file> on the"
```

Comment:
Drop reference to `--config` as that is not an option.

Reply:
Are you going to make this change?

Reply:
She did...are you looking at the current sources?